Deployment/scripts/deployAzureResources.ps1

```
@@ -519,104 +543,292 @@ try {
+    if ([string]::IsNullOrEmpty($json)) {
+        throw [System.Exception]"NOLOGIN"
+    }
+    Write-Host "$($json | ConvertFrom-Json | ConvertTo-Json)" -ForegroundColor Green
+    deploy_main_services
+
+    # Step2
+    get_service_info
+
+    # Step3
+    update_app_configs
+
+    # Step4
+    build_push_container_images
+
+    # Step5
+    configure_k8s
+    upgrade_k8s
+
+    # Step6
+    ### REMOVED
+
+    # Step7
+    configure_aks
+
+    # Step8
+    closing_remarks
+
+} catch {
+    Write-Host "$($MyInvocation.MyCommand.Name): Error in main" -ForegroundColor Red
     Write-Host $_.Exception.Message -ForegroundColor Red
-    Write-Host $_.Exception.StackTrace -ForegroundColor Red
+    Write-Host $_.InvocationInfo.PositionMessage -ForegroundColor Red
+    Write-Host $_.ScriptStackTrace -ForegroundColor Red
+} finally {
+    Remove-Variable LOG -ErrorAction SilentlyContinue
+    Remove-Variable LOGDIR -ErrorAction SilentlyContinue
+    Remove-Variable RESULTS_OUT -ErrorAction SilentlyContinue
+    Remove-Variable CWD -ErrorAction SilentlyContinue
+    Remove-Variable deploymentResult -ErrorAction SilentlyContinue
+    Remove-Variable iac_dir -ErrorAction SilentlyContinue
+    Remove-Variable is_testing -ErrorAction SilentlyContinue
+    Remove-Variable msg -ErrorAction SilentlyContinue
+    Remove-Variable STAMP -ErrorAction SilentlyContinue
+    Remove-Variable TIMESTAMP -ErrorAction SilentlyContinue
+
+    Remove-Variable subscriptionID -ErrorAction SilentlyContinue
+    Remove-Variable location -ErrorAction SilentlyContinue
+    Remove-Variable email -ErrorAction SilentlyContinue
+    Remove-Variable ipRange -ErrorAction SilentlyContinue
+    Remove-Variable prefix -ErrorAction SilentlyContinue
+    Remove-Variable appname -ErrorAction SilentlyContinue
+    
+    Remove-Variable json -ErrorAction SilentlyContinue
+
+    Write-Host "Script complete $(Get-Date -Format 'yyyyMMdd_T_HHmmss')" -ForegroundColor DarkYellow
+    Stop-Transcript
```

We are getting an error because Stop-Transcript is being called when no transcription is active. Please add a check to ensure it's only called when transcription is started.

---

Added try catch to squash error